src/actions/guides/database/database_setup.cr

```diff
@@ -49,6 +49,55 @@ class Guides::Database::DatabaseSetup < GuideAction
     This causes Lucky to raise an exception if you forget to preload an association,
     but will not raise an exception in production.
 
+    ## Multiple Databases
```

Comment:
I think this should go near the bottom as it is probably the least relevant for most people.